packages/darklang/languageTools/writtenTypes.dark

```diff
@@ -5,6 +5,7 @@ module Darklang =
       type SourceRange = Parser.Range
       // <aliases>
 
+      type Name = Unresolved of List<String>
```

Comment:
Do we need this in WrittenTypes? I don't think it's referenced in the other types here, and realistically (almost) _all_ names in WrittenTypes are sort of unresolved.

Maybe it belongs in PT.dark or WT2PT.dark. Is the intent to capture the fact that a name may be 'unresolved' during the "first pass" of name resolution?

Reply:
Yes, the only use for it was to capture the unresolved name in the name resolver.